01-Introducing-Molecular-Dynamics/01-Molecular-Dynamics-Simulations.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>It may not be immediately clear why we use the <strong>NVT</strong> rather than the <strong>NVE</strong> ensemble for the LJ crystallization. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I added a sentence that provides a reason.</p>